README.md

```diff
@@ -1,591 +1,51 @@
-# Giskard
+<p align="center">
+  <img width="300" alt="giskardlogo" src="frontend/src/assets/logo_full.svg">
+</p>
 
-## Backend Requirements
+<h3 align="center">Quality Assurance for AI</h3>
+<br />
+<p align="center">
+    <img src="readme/inspect.png" alt="Administration panel" />
+</p>
 
-* [Docker](https://www.docker.com/)
-* [Docker Compose](https://docs.docker.com/compose/install/)
-* [Poetry](https://python-poetry.org/) for Python package and environment management
+<br/>
 
-## Frontend Requirements
+Giskard is an open-source quality assurance platform for AI models.
 
-* Node.js (with `npm`).
+- **Inspect** python models using visual interface
```

Comment:
These 3 bullet points may need a bit of re-wording to make them more impactful. 

Ex: 
- **Inspect** ML models visually from your Python notebook 📗
- **Review** ML models collaboratively to get feedback on potential errors 🧐
- **Test** ML models automatically for a safe deployment to production ⛑

WDYT @jmsquare ?

Reply:
I like this version, I pushed it